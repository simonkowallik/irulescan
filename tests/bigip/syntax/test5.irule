when RULE_INIT priority 500 {
switch    {4}{  
  {4}{log local0. [expr 4]} }   
}