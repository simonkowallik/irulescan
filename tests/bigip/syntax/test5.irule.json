[
  {
    "filepath": "test5.irule",
    "warning": [
      "Unquoted expr at `4` in `expr 4`"
    ],
    "dangerous": [
      "missing options terminator `--` permits argument injection at `{4}` in `switch    {4} {   {4} {log local0. [expr 4]} }   `"
    ]
  }
]
