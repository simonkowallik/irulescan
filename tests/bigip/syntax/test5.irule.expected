DANGEROUS: missing options terminator `--` permits argument injection at `{4}` in `switch    {4} {   {4} {log local0. [expr 4]} }   `
WARNING: Unquoted expr at `4` in `expr 4`

